packages/terra-form/src/Textarea.scss

```diff
@@ -1,14 +1,78 @@
 :local {
   .textarea {
-    border: #dedfe0 solid 1px;
-    border-radius: 0.25em;
-    font-size: 1em;
-    line-height: 1.25;
-    padding: 0.178em;
+    background-color: var(--terra-form-textarea-background-color, #fff);
+    background-image: var(--terra-form-textarea-focus-animation-background, linear-gradient(to bottom, #396ee5, #396ee5));
```

Comment:
Should we really have this gradient set as the default? I would expect the default value for background-image to be none.